.travis/install.sh

```diff
@@ -4,8 +4,6 @@ set -e
 set -x
 
 if [[ "$(uname -s)" == 'Darwin' ]]; then
-    brew update || brew update
```

Comment:
hmm, i also wondered whether travis is running some brew update in the background?
sometimes seen a message about it already running.
but how are tests supposed to be stable if there is a upgrade running in the background?